HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -1017,7 +1017,7 @@ def initialize(name, used_by_unavailable_periods, can_be_stochastic, type)
     Battery: Column.new('battery', false, false, :neg_one_to_one),
     BatteryCharging: Column.new('battery_charging', true, false, nil),
     BatteryDischarging: Column.new('battery_discharging', true, false, nil),
-    ElectricVehicle: Column.new('electric_vehicle', false, false, :neg_one_to_one),
+    ElectricVehicle: Column.new('electric_vehicle', false, true, :neg_one_to_one),
```

Comment:
Bugfix. Without this change, the `electric_vehicle` column would no longer be exported.